client/src/utils/clipboard.js

```diff
@@ -4,7 +4,7 @@ import { Toast } from "ui/toast";
 export function copy(text, notificationText) {
     if (navigator.clipboard) {
         navigator.clipboard.writeText(text).then(() => {
-            Toast.info(notificationText);
+            if (notificationText) Toast.info(notificationText);
```

Comment:
Should this be
```suggestion
            if (notificationText) {
                Toast.info(notificationText);
            }
```
? It works, but I don't think we use this commonly across the codebase, not sure we should start doing it now.

I'm also curious if we should even write to the clipboard if there is no text ?

Reply:
> I'm also curious if we should even write to the clipboard if there is no text ?

This is not the text that goes into clipboard. It's just the text that appears in a Toast, if someone doesn't want to see a Toast `notificationText` will be empty.

> It works, but I don't think we use this commonly across the codebase, not sure we should start doing it now.

The behavior of `if()` is defined in our eslint. The rules tell us, if it doesn't have curly brackets it should be one line. If you don't like it, it's ok, I will change it here. But in general, if we don't support it, we should change the config of eslint. It's in the code and I used it in many places already.